src/__test__/actions/signupActions.test.js

```diff
@@ -0,0 +1,83 @@
+import configureMockStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
+import moxios from 'moxios';
+
+import signupAction from '../../actions/signupAction';
+import { REGISTER_USER, REGISTER_USER_SUCESS, REGISTER_USER_FAILURE } from '../../actions/types';
```

Comment:
If REGISTER_USER_FAILURE isn't being used please remove it

Reply:
@NicholusMuwonge this has been worked on